etc/input.conf

```diff
@@ -67,7 +67,7 @@
 #{ multiply speed 0.5
 #} multiply speed 2.0
 #BS set speed 1.0                       # reset speed to normal
-#q quit
+#Ctrl+q quit
```

Comment:
This really is personal preference, so I don't think the default should be changed, especially since you'll annoy all existing users who use the defaults as-is.

I for one enjoy having q be quit.